tests/test_serializers.py

```diff
@@ -178,9 +178,12 @@ class TestNodeLogSerializers(OsfTestCase):
 
     def test_serialize_log(self):
         node = NodeFactory(category='hypothesis')
-        log = NodeLogFactory(params={'node': node._primary_key})
-        node.logs.append(log)
         node.save()
+        log = NodeLogFactory(
+                params={'node': node._primary_key},
```

Comment:
Just for consistency could you mind changing this to `node._id`? `_primary_key` is an alias to `_id` for internal use in ODM, which is why it's prefixed with a `_`.  `_id` is prefixed because mongodb has decided to prefix it.